pkgs/tools/package-management/nix/default.nix

```diff
@@ -235,7 +239,7 @@ in lib.makeExtensible (self: ({
 
   stable = addFallbackPathsCheck self.nix_2_18;
```

Comment:
```suggestion
  stable = addFallbackPathsCheck self.nix_2_19;
```

Reply:
We can bump stable in a bit after determining that things are working as they should.